scripts/Units/armliche.bos

```diff
@@ -124,43 +135,32 @@ Activate()
 
 Deactivate()
 {
+	active = 7;
 	hide thrustc;
 	hide thrusta;
 	hide thrustb;
 	start-script RequestState(1);
 }
 
-FirePrimary()
-{
-	if( gun_1 )
-	{
+FirePrimary() {
 	hide bomb1;
-	}
-	else
-	{
-	hide bomb2;
-	}
-	
-	gun_1 = !gun_1;
+	start-script ReloadBomb();
+	return (1);
 }
 
 AimPrimary()
 {
-	if( gun_1 )
-	{
-	show bomb1;
-	}
-	else
-	{
-	show bomb2;
+	if (active < (get(GAME_FRAME) - 60)) {
+		
+		return (1);//only shoot if we've been flying for more than a second
+	} else {
+		return (0);
```

Comment:
This would take a more experienced *bos*s than myself, but I think the gif you made for this would go great as an explanation for the change :)